tests/lib/snaps/basic18/meta/snap.yaml

```diff
@@ -0,0 +1,5 @@
+name: basic18
```

Comment:
Adding a comment here since github won't let me comment on the binary icon file: I think the icon.png is not relevant for this test, so perhaps it could be dropped?

Reply:
I will drop those icons from gadgets in a follow up